build/common.ps1

```diff
@@ -227,6 +227,7 @@ Function Install-DotnetCLI {
     # However, Invoke-BuildStep checks if any error happened, ever. Hence we need to run dotnet-install
     # in a different process, to avoid treating their handled errors as build errors.
     & powershell $DotNetInstall -Runtime dotnet -Channel 3.1 -InstallDir $CLIRoot -NoPath
+    & powershell $DotNetInstall -Runtime dotnet -Channel 5.0 -InstallDir $CLIRoot -NoPath
```

Comment:
Why do we need this?

Reply:
I started to not be able to run unit tests locally because .NET 5 is removed by Visual Studio 2022 17.4 since its EOL.  Our unit tests are generally built for `netcoreapp3.1`, `net5.0` and `net472` so I figured we might as well install the `net5.0` runtime since we're already installing `netcoreapp3.0`.  Should I do this in different change?

Reply:
Doesn't that only install the runtime for the test SDK.
As in, locally I expected your project to be using the machine SDK and runtime.

Reply:
Ah yes that would explain it.  I'm fine with reverting this for this PR, its up to you.